multicast/send.py

```
@@ -277,27 +277,27 @@
+				"--group",
 				default=multicast._MCAST_DEFAULT_GROUP  # skipcq: PYL-W0212 - module ok
 			)
 			parser.add_argument(
-				"""--groups""",
+				"--groups",
 				required=False,
 				nargs="*",
-				dest="""groups""",
-				help="""multicast groups (ip addrs) to listen to join."""
+				dest="groups",
+				help="multicast groups (ip addrs) to listen to join."
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)